Makefile

```diff
@@ -194,7 +194,10 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 
 .install.md2man: .gopathok
 	if [ ! -x "$(GOPATH)/bin/go-md2man" ]; then \
-		go get -u github.com/cpuguy83/go-md2man; \
+		   go get github.com/cpuguy83/go-md2man; \
```

Comment:
Use `-d` flag here so we stop after downloading and don't install the bad version